components/next_steps_view/next_steps_tips.tsx

```diff
@@ -309,6 +314,63 @@ export default function NextStepsTips(props: Props) {
         );
     }
 
+    let channelsSection;
+    if (props.isFirstAdmin) {
+        channelsSection = (
```

Comment:
Could just create two tiny components, would be much easier to read the whole thing as opposed to this giant if/else block :)

Reply:
I think this is a good idea, but want to do it to the entire component since all of these tips are just a card with title + description + action button. 

I've simplified the if statements to set an object that renders the one card instead of 2 possible JSX fragments. This is much more readable in the interim. 

I am going to open a ticket to actually refactor the whole page into using something similar to the [TextCardWithAction](https://github.com/mattermost/mattermost-webapp/tree/master/components/next_steps_view/steps/text_card_with_action) component I built for the next_steps_view

Reply:
Sounds good! Thanks for making the change @nickmisasi! 🎉